### PR TITLE
Add the baseband python package

### PR DESCRIPTION
**Pull request by mwcraig**
This pull request adds the astropy affiliated package baseband. The patch is necessary because the astropy infrastructure currently tries to auto-update astropy-helpers during install.

This will eventually change upstream.

### CODE REVIEW

**Comment by mwcraig** [comment]
A PR has been opened upstream to add the license to the source distribution: https://github.com/mhvk/baseband/pull/318

File: recipes/baseband/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - baseband.mark4
+    - baseband.mark5b
+    - baseband.vdif
+    - baseband.vlbi_base
+
+about:
+  home: https://baseband.readthedocs.io
+  license: GPLv3
+  license_family: GPL3
+  license_file: {{ RECIPE_DIR }}/LICENSE
```
Comment (chrisburr) [inline recipes/baseband/meta.yaml]: Conda will automatically look in the recipe directory if it isn't found in the build directory:
```suggestion
  license_file: LICENSE
```

File: recipes/baseband/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - baseband.guppi
+    - baseband.helpers
+    - baseband.mark4
+    - baseband.mark5b
+    - baseband.vdif
+    - baseband.vlbi_base
+
+about:
+  home: https://baseband.readthedocs.io
+  license: GPLv3
```
Comment (chrisburr) [inline recipes/baseband/meta.yaml]: From [here](https://opensource.org/licenses/alphabetical)
```suggestion
  license: GPL-3.0
```

File: recipes/baseband/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  license_family: GPL3
```
Comment (chrisburr) [inline recipes/baseband/meta.yaml]:
```suggestion
  license_family: GPL
```

**Comment by mwcraig** [comment]
Thanks for the review @chrisburr. The entry point has been fixed in this upstream PR: https://github.com/mhvk/baseband/pull/317

**Comment by chrisburr** [comment]
Thanks for making the PR to fix the conda-forge docs, LGTM! 👍

**Pull request merged**